### PR TITLE
Hotfix az rewrite and storage_preview

### PR DESCRIPTION
**Pull request by haroldrandom**
* Hotfix: bug introcuded by azure-cli commit [155b6bc Rewrite az in Python](https://github.com/Azure/azure-cli/commit/155b6bc1f9e358757c629cddd69bc3f0862f50cc)
* Update wrong sha256Digest of storage_preview
* Disallow failures on Travis CI

**WARNING**: This hotfix comments out  `azdev verify load-all` and `azdev cli-linter` (from azure-cli) , which opens the Pandora box that leave extension commands without some correctness verification. Will migrate those function into the public [azure-cli-dev-tools](https://github.com/Azure/azure-cli-dev-tools) ASAP.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/index.json [hunk 1]
```diff
@@ -2218,7 +2218,7 @@
                     "summary": "Provides a preview for upcoming storage features.",
                     "version": "0.2.9"
                 },
-                "sha256Digest": "211afaf23a6340671fee8a4b9e7c982f28b9ec9d1bf2e403fcc9de08c561ab7b"
+                "sha256Digest": "880e01de0fab8893770497ef9410559ae223a1f09dbd6a23712226ab4e2d5ecb"
```
Comment (Juliehzl) [inline src/index.json]: You also change index.json for storage-preview?

File: scripts/ci/verify_modified_index.sh [hunk 2]
```diff
@@ -26,17 +28,19 @@ while read line; do
...
-    echo "Running linter"
-    azdev cli-lint --ci --extensions $ext
+
+    # TODO migrate to public azdev
+    echo "Load all commands..."
+#    azdev verify load-all
+
+    # TODO migrate to public azdev
+    echo "Running linter..."
+#    azdev cli-lint --ci --extensions $ext
```
Comment (MyronFanQiu) [inline scripts/ci/verify_modified_index.sh]: should we run the azdev linter and azdev verify load-all?

File: src/index.json (hunk 1 above)
Comment (haroldrandom) [reply src/index.json]: resolved offline.

File: .travis.yml [hunk 3]
```diff
@@ -50,5 +50,3 @@ jobs:
       env: PURPOSE='SyncAvailableExtensionsDoc'
       if: repo = Azure/azure-cli-extensions and branch = master and type = cron
   fast_finish: true
-  allow_failures:
```
Comment (MyronFanQiu) [inline .travis.yml]: Is this intentional?

Comment (haroldrandom) [reply .travis.yml]: https://github.com/Azure/azure-cli-extensions/issues/1067  by convenience. Leave the open will introduce more confusion.

File: scripts/ci/verify_modified_index.sh (hunk 2 above)
Comment (haroldrandom) [reply scripts/ci/verify_modified_index.sh]: Currently, the public [azdev](https://github.com/Azure/azure-cli-dev-tools) doesn't have those two commands. So it's a dangerous hotfix which will open the Pandora Box that leave extension commands without some correctness verification.

**Pull request merged**